web-share/share-extra-argument-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Surplus arguments ignored</title>
```

Comment:
I don't think you want to test for this. This might cause issues if we need to extend the API later and it's outside the scope of the API.